jira.py

```diff
@@ -0,0 +1,37 @@
+import os
+import sys
+
+from lib.jira_conn import JiraAPIClient
+
+
+# JQL query All QA Requests since 2022 filter_id: 13856
+FILTER_ID_ALL_REQUESTS_2022 = "13856"
+# Fields needed
+STORY_POINTS = "customfield_10037"
+FIREFOX_RELEASE_TRAIN = "customfield_10155"
+ENGINEERING_TEAM = "customfield_10134"
+DEFAULT_COLUMNS = "id,key,status,created,summary,labels,"
+MAX_RESULT = "maxResults=100"
+
+JQL_QUERY = 'jql=filter='
+
+
+class Jira:
+
+    def __init__(self):
+        try:
+            JIRA_HOST = os.environ['JIRA_HOST']
+            self.client = JiraAPIClient(JIRA_HOST)
+            self.client.user = os.environ['JIRA_USER']
```

Comment:
I assume the Jira API has more than one authentication scheme: username/pw, Oauth, Token, etc.  Probably doesn't matter which one we use but we should ensure we're using a service account here

Reply:
yes, we are using a service account here. The one we have stored in 1Password